pkgs/tools/text/all-packages.nix

```diff
@@ -0,0 +1,3 @@
+{ callPackage }: {
+  markitdown = callPackage ./markitdown { };
+}
```

Comment:
yeah, we don't do this anymore. The markitdown file should be put in `by-name` instead.